.github/workflows/build_push_update_cal.asgn.yml

```diff
@@ -31,14 +32,15 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
         contents: 'write'
+        security-events: write
+        packages: write
         id-token: 'write'
         pull-requests: 'write'
-
     steps:
       - name: "Checkout"
         uses: "actions/checkout@v4.1.4"
         with:
-          ref: 'master'
+     #     ref: 'master'
```

Comment:
This should probably still only work on master. Commented out for testing?

Reply:
It may work when it is pushed to master, When uncommented, it errored out for some nonsense reason. I will keep an eye on when merged to master.